tensorflow/python/lib/io/file_io.py

```diff
@@ -246,6 +246,10 @@ def close(self):
         pywrap_tensorflow.Set_TF_Status_from_Status(status, ret_status)
     self._writable_file = None
 
+  def seekable(self):
```

Comment:
This should be a @property

Reply:
Actually it should not be a property, because [`zipfile` calls `fileobj.seekable()`](https://github.com/python/cpython/commit/066df4fd454d6ff9be66e80b2a65995b10af174f#diff-30e3ce7dc73ea0f34b323c729c652a08R798). This code still doesn't work. :sob:

Could you fix this problem with other PR again? @yongtang